module/mailnews/newsread.c

```diff
@@ -190,48 +201,48 @@ BOOL CALLBACK ReadNewsDialogProc(HWND hDlg, UINT message, UINT wParam, LONG lPar
       /* Get rid of old index, if any */
       if (info->articles != NULL)
          list_destroy(info->articles);
-      
-      info->articles = (list_type) lParam;
-      
+
+      info->articles = (list_type)lParam;
+
       /* Put article title lines in list box; save old position if appropriate */
       index = -1;
       if (ListView_GetItemCount(hList) > 0)
-         if (!ListViewGetCurrentData(hList, &index, (int *) &article))
+         if (!ListViewGetCurrentData(hList, &index, (int *)&article))
             index = -1;
       ListView_DeleteAllItems(hList);
       for (l = info->articles; l != NULL; l = l->next)
       {
-         article = (NewsArticle *) (l->data);
-         
+         article = (NewsArticle *)(l->data);
+
          // Add article to list view
          if (!IsNameInIgnoreList(article->poster))
          {
             lvitem.mask = LVIF_TEXT | LVIF_PARAM;
             lvitem.iItem = ListView_GetItemCount(hList);
             lvitem.iSubItem = 0;
             lvitem.pszText = article->title;
-            lvitem.lParam = (LPARAM) article;
+            lvitem.lParam = (LPARAM)article;
```

Comment:
These little whitespace changes obscure what this change is about (and actually I disagree with deleting the space here).  Assuming this is some kind of auto-linter in your editor, can you turn it off?

Reply:
I'll have to dig around and see. I'm running Visual Source Code's default code style formatting. Probably not a great idea w/o a full understanding of the desired code style, but I'm running into a lot of difficult to read blocks of code and have been using it sparingly to make the code more readable.

Reply:
I found myself adopting the use of `clang-format` and curious if there'd be interest in adopting it here as well? We could add a root level `.clang-format` and throw in [everything and anything](https://clang.llvm.org/docs/ClangFormatStyleOptions.html) that we'd like to work towards.

Reply:
Hm, maybe, but certainly not in this change.